api/README.md

```diff
@@ -135,6 +135,9 @@ curl 'http://myAdminUser:myAdminPass@localhost:5984/medic/_design/medic-client/_
 
 So, if you want to re-run a migration, delete its entry in the `migrations` list and re-run api.
 
+## Troubleshooting
```

Comment:
I think this is a great addition, but should be in the main README, because that's where people will encounter this (i.e, if it happens it's going to happen the very first time they boot api)